pyvista/errors.py

```diff
@@ -0,0 +1,9 @@
+"""Pyvista specific errors."""
```

Comment:
The only way I could figure out how to avoid circular imports was to create a new module here.  IMO, most of the errors in `pyvista.core.errors` are not related to `pyvista.core` and could be moved up.  I'm not sure if that would be a breaking change, however.

Reply:
I guess we could try providing deprecated aliases in `pyvista.core` assuming we can get around the circular import problem.

Reply:
Let's leave out from this PR moving and deprecating the existing errors in `pyvista.core.errors`.  I mainly brought this up to justify making my life easier to create a new module here 😄 .